omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/DiagnosesFragmentController.java

```diff
@@ -105,7 +105,7 @@
             }
         }
         else {
-            sources = emrApiProperties.getConceptSourcesForDiagnosisSearch();
+            sources = StringUtils.isNotEmpty(diagnosisSetUuids) || StringUtils.isNotEmpty(diagnosisConceptClasses) ? null : emrApiProperties.getConceptSourcesForDiagnosisSearch();
```

Comment:
For the sake of this controller, should the check not be
```java
diagnosisSetUuids != null || diagnosisConceptClasses != null
```
?

Reply:
The same story as above happens here @mks-d.